internal/armada/server.go

```diff
@@ -476,10 +483,73 @@ func createRedisClient(config *redis.UniversalOptions) redis.UniversalClient {
 	return redis.NewUniversalClient(config)
 }
 
-// TODO Is this all validation that needs to be done?
-func validateArmadaConfig(config *configuration.ArmadaConfig) error {
+// TODO: Is this all validation that needs to be done?
+func validateCancelJobsBatchSizeConfig(config *configuration.ArmadaConfig) error {
```

Comment:
I've renamed this function because it claimed to validate all the armada config but all  it actually did was check that the `cancelJobsBatchSize` was greater than 0!  Clealry we need to have a better strategy for validating our configuration but that's a job for another day.